server/src/test/java/org/apache/druid/client/CachingClusteredClientTest.java

```diff
@@ -2213,13 +2213,13 @@ public void run()
             expectedResults.get(k).get(j)
         );
         serverExpectations.get(lastServer).addExpectation(expectation);
-
+        EasyMock.expect(mockSegment.getSize()).andReturn(-1L).anyTimes();
```

Comment:
I guess you're simulating the segment from realtime tasks. Then, it would be more realistic if it returns 0.

Reply:
okay, changed to 0